packages/core/utils/lib/convert-query-params.js

```diff
@@ -147,18 +147,72 @@ const convertPopulateQueryParams = (populate, depth = 0) => {
   }
 
   if (_.isPlainObject(populate)) {
-    const transformedPopulate = {};
-    for (const key in populate) {
-      transformedPopulate[key] = convertNestedPopulate(populate[key]);
-    }
-
-    return transformedPopulate;
+    return convertPopulateObject(populate, schema);
   }
 
   throw new InvalidPopulateError();
 };
 
-const convertNestedPopulate = subPopulate => {
+const convertPopulateObject = (populate, schema) => {
+  if (!schema) {
+    return {};
+  }
```

Comment:
It shouldn't be possible, I can remove that check so that it's easier to debug when it'll happen. Also I can log a warning and pass down the empty object, what do you think @petersg83 ?